src/blocks/weather/met_no.rs

```diff
@@ -284,7 +284,9 @@ impl WeatherProvider for Service<'_> {
         };
 
         Ok(WeatherResult {
-            location: "Unknown".to_string(),
+            location: location
+                .as_ref()
```

Comment:
I'm surprised clippy doesn't catch this, but this `.as_ref()` is not needed.

Reply:
I suppose most `Option<&T>::as_ref()` calls are useless, except for rare cases when `Option<&&T>` is actually needed. I may look into adding this to clippy in the future.